src/contracts/pods/EigenPod.sol

```diff
@@ -605,22 +600,23 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     }
 
         /// @notice payable fallback function that receives ether deposited to the eigenpods contract
-    function receive() external payable {
+    function receiveETH() external payable {
```

Comment:
yeah, def want to revert this change, it should be called `receive` to work as a payable fallback function

Reply:
@Sidu28 looks like you just deleted this instead? just trying to figure out what you're trying to do in this PR, frankly.